prow/metrics/http.go

```diff
@@ -71,6 +81,16 @@ func histogram(name, help string, buckets []float64) *prometheus.HistogramVec {
 	)
 }
 
+func counter(name, help string) *prometheus.CounterVec {
```

Comment:
Why does this func exist, it has exactly one caller and no dedicated tests?

Also, the name is vastly misleading, this is not a `counter` its a `counterVec` and its only suited for the one caller that uses it (because it has the allowed label already configured) so I'd prefer to inline this into that caller.

Reply:
I added that to stay consistent with the histogram function which was being used for HTTPResponseSize and RequestDuration. I can inline it..